resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml

```diff
@@ -70,7 +70,7 @@ Cura.ExpandablePopup
                         font: UM.Theme.getFont("default")
                         color: UM.Theme.getColor("text")
                         renderType: Text.NativeRendering
-
+                        opacity: model.enabled ? 1 : 0.4
```

Comment:
Maybe this would be better done through a theme file, so that it can be specialised for e.g. the high contrast themes?

I consider this to be subjective though, so I won't let it block the issue and continue testing.